server/src/main/java/io/crate/execution/dsl/projection/SourceIndexWriterReturnSummaryProjection.java

```diff
@@ -100,6 +110,7 @@ public void writeTo(StreamOutput out) throws IOException {
         Symbols.toStream(sourceUri, out);
         Symbols.toStream(sourceUriFailure, out);
         Symbols.toStream(lineNumber, out);
+        Symbols.toStream(sourceParsingFailure, out); // This field was introduced in 4.7.2
```

Comment:
There needs to be a version check here too - could be sending it to an older node

Reply:
https://github.com/crate/crate/pull/12229/commits/61d5b9ff921f756b13ceba46b52b85455392e214